Framework/Algorithms/src/WorkspaceJoiners.cpp

```diff
@@ -155,7 +157,7 @@ MatrixWorkspace_sptr WorkspaceJoiners::execEvent() {
     m_progress->report();
   }
 
-  fixSpectrumNumbers(*event_ws1, *event_ws2, *output);
+  fixSpectrumNumbers(event_ws1, event_ws2, *output);
```

Comment:
While this is being modified it is probably worth replacing all the `snake_case` variable names with `lowerCamalCase`.